modules/resetdigitalBidAdapter.js

```diff
@@ -43,16 +43,29 @@ export const spec = {
       };
     }
 
+    if (bidderRequest && bidderRequest.uspConsent) {
+      payload.ccpa = bidderRequest.uspConsent;
```

Comment:
Your title says gdpr but your code doesn't?

Reply:
gdpr it's on line 39

Reply:
right but that is already there, it isnt changing.